src/api/crosscutRequests.js

```diff
@@ -95,4 +96,30 @@ export const deleteCatchmentJob = async (id) => {
     } catch (err) {
         throw err
     }
+}
+
+export const fetchSupportedBoundaries = async () => {
+    const boundaryVerion = "v3"
+    const url = `${baseURL}/boundaries/${boundaryVerion}`
+    try {
+        const resp = await ky(url, {
+            mode: "cors",
+            headers: {
+                authorization: getToken(),
+            }
+        }).json()
+
+        resp.boundaryList.forEach((bound) => {
+            if (bound.areaName === "" && bound.entireCountry === true) {
+                bound.areaName = i18n.t("entire country")
+            }
```

Comment:
For the moment, do we only want to include entire countries in the drop-down list? Will we be able to support partial countries before https://github.com/crosscutio/dhis2-crosscut-app/issues/9 is complete?

Reply:
A user should be able to create catchments as they will receive errors on the sites that fall outside of our boundary and create the catchment with the rest of the sites.